packages/styletron-react/src/index.js

```diff
@@ -11,7 +11,7 @@ import {
   driver,
   getInitialStyle,
   type StandardEngine,
-  type StyleObject,
+  type StyleObject as StdStyleObject,
```

Comment:
Won't this break existing usages of `StyleObject`?
```suggestion
  type StyleObject,
```

Reply:
Nevermind, type declarations are hoisted. But I still think we can just export the actual name